tests/cypress/plugins/unpackZipArchive/addPlugin.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+// eslint-disable-next-line no-use-before-define
+exports.unpackZipArchive = unpackZipArchive;
```

Comment:
Please do not disable rules if they can be fixed easily. Just push exports after the function declaration.

Reply:
Fixed.